README.md

```diff
@@ -43,6 +43,7 @@ Span #19
 
 ```bash
 OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318" bin/rails server
+OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=http/json
```

Comment:
This is added to our Kubernetes cluster in this PR: https://github.com/actblue/infrastructure/pull/1439